client/cmd/auth.go

```diff
@@ -241,13 +242,16 @@ func Cancel(username string, password string, yes bool) error {
 	}
 
 	err = auth.Delete(c, username)
-
-	if err != nil {
+	cleanup := fmt.Errorf("\n%s %s\n\n", "409", "Conflict")
+	if reflect.DeepEqual(err, cleanup) {
```

Comment:
@helgi we talked about this offline, but capturing here. I'd like `auth.Delete` to put this error string in its API contract, or better yet have it return a specific, exported error (like `409Error` or something), so that we can get stronger compile time checks